### ISSUE TITLE
function for RESTful GET parameter `limit` is missing

### ISSUE THREAD

**Issue by laura-ham** [semi-technologies/weaviate-javascript-client#9]
`withLimit()` does work with `.graphql.get` functions, but not for the `.getter` function. This is expected to work:

```js
const weaviate = require("weaviate-client");

const client = weaviate.client({
  scheme: 'http',
  host: 'localhost:8080',
});

client
    .data
    .getter()
    .withLimit(2)  // THIS IS CURRENTLY NOT IMPLEMENTED 
    .withUnderscoreClassification()
    //.withUnderscoreInterpretation()
    //.withUnderscoreNearestNeighbors()
    //.withUnderscoreFeatureProjection()
    //.withUnderscoreVector()
    .do()
    .then(res => {
        console.log(res)
    })
    .catch(err => {
        console.error(err)
    });
```

**Comment by etiennedi** [comment]
Should be fixed/implemented in v1.0.0-rc4. Keeping this issues open until I have feedback that it works as desired.

**Comment by etiennedi** [comment]
Confirmed working by @laura-ham. Closing.
